native/packages/react-native-bpk-component-icon/src/BpkIcon.js

```diff
@@ -32,12 +32,11 @@ const styles = StyleSheet.create({
   icon: {
     fontFamily: 'BpkIcon',
     fontSize: spacingLg,
-    lineHeight: spacingLg,
```

Comment:
Do we not still need this to ensure we get the exact height we want or is that embedded in the font?

Reply:
Please refer to the images in the desc of this PR. 

The dimensions are now 24 x 24 as expected as a result of removing the line height.